test/parallel/test-cluster-disconnect.js

```diff
@@ -47,6 +47,7 @@ if (cluster.isWorker) {
       // check result
       socket.on('end', common.mustCall(() => {
         cb(result === 'echo');
+        server_ports.splice(server_ports.indexOf(port));
```

Comment:
You're absolutely right. Thanks for pointing that out. I missed it somehow. I've pushed the changes.